mobile/src/main/java/ch/epfl/sdp/mobile/ui/features/social/ChessMatch.kt

```diff
@@ -0,0 +1,18 @@
+package ch.epfl.sdp.mobile.ui.features.social
+
+class ChessMatch(
+    val adv: String,

```

Comment:
Might be worth naming `adversary` instead.

Reply:
👍